egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+num_epochs=100
```

Comment:
For diagnostic purposes you should probably maintain a small held-out subset of data for validation (e.g. the smaller of, 1000 utterances or 5% of the data).  After a while you expect it to start getting worse on validation data.   So convergence on the training set may not be the best criterion.
But I suggest to just specify the number of epochs in advance-- supply a default but make it optional.